test/WebSites/BasicWebSite/Controllers/OrderController.cs

```diff
@@ -19,7 +19,7 @@ public int GetServiceOrder(string serviceType, string actualType)
             var services = (IEnumerable<object>)HttpContext?.RequestServices.GetService(queryType);
             foreach (var service in services)
             {
-                if (actualType != null && service.GetType().AssemblyQualifiedName == actualType)
+                if (actualType != null && service.GetType().FullName == actualType)
```

Comment:
Unrelated bug?

Reply:
I don't want to pass the full name with version along. Anyway, this test should go away soon.